kernel-laws/shared/src/test/scala/cats/kernel/laws/LawTests.scala

```diff
@@ -66,8 +66,22 @@ object KernelCheck {
   }
 
   // Copied from cats-laws.
-  implicit def arbitrarySortedSet[A: Arbitrary: Order]: Arbitrary[SortedSet[A]] =
-    Arbitrary(arbitrary[Set[A]].map(s => SortedSet.empty[A](implicitly[Order[A]].toOrdering) ++ s))
+  implicit def arbitrarySortedSet[A: Arbitrary: Order: Cogen]: Arbitrary[SortedSet[A]] = {
```

Comment:
you aren't using `Order[A]` any more are you?

Reply:
It looks like I will need it per @joroKr21's comment above to hit the fast path branch in testing.

https://github.com/typelevel/cats/pull/4104#discussion_r777611387